docs/pages/docs/docs-theme/theme-configuration.mdx

```diff
@@ -483,6 +483,26 @@ export default {
 }
 ```
 
+## Theme Switch
+
+You are able to customize the option names:
+
+```jsx
+export default {
+  themeSwitch: {
+    options() {
+      //hooks
+      
+      return [
+        { key: 'light', name: 'Light' },
```

Comment:
That's not the way, what if user returns only one item or two? makes no sense to use a select anymore

still, my initial suggestion

```
themeSwitch: {
  labels: ({ locale }) => ({
    toggle: "Toggle theme",
    system: "With system",
    dark: "Dark mode",
    light: "Light mode",
  })
}
```
and if we want to keep consistency with other API, then rename it to `useLabels`

Reply:
Those are accessible labels, and internally we will use it as `const options = [{key: 'system', label: labels.system...`
I really don't think user need to provide the full options, unless you support more themes, from the codebase, Nextra supports 3 modes, and can't be changed (e.g. I don't need `system` but I can't opt it out)

Reply:
And even we support customise modes, like I only need two modes(light/dark), then I don't need a select at all, a toggle icon button will do the job and that's how it looks in most websites, then what's the point of `options`?

Reply:
@B2o5T please provide more constructive comments

Reply:
I don't disagree that using an object instead of array might be better

Support for multi-themes or disabling `system` theme is unrelated to this pull request, it's just basically a config of the component, please override the next-themes configuration or open a feature request for that

My suggestion is to add a `component` property for overriding the whole component, similar to others

Reply:
A good API is hiding internal details, by adding `options`, you made *visible* modes having accessible labels but what about the toggle button? Exposing another prop? It's also important for A11y

Reply:
The theme switch is a Select component based on headless ui, What did you mean by the toggle button?

Reply:
the select trigger...

Reply:
ok I got you, the localisation for aria-label isn’t supported by Nextra (ex: Sidebar toggle) 
It’s weird to only support this feature in theme switch

Reply:
> the localisation for aria-label isn’t supported by Nextra (ex: Sidebar toggle)
It’s weird to only support this feature in theme switch

So apparently you agree it's wrong, so why not do it right here, and we can address the issue in other places